scripts/ci/testing/run_breeze_command_with_retries.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This script should be only used commands without waiting interactive inputs.
+# For example: 'apt-get install [package_name]' will ask you to input 'y' while installing.
+# This is an interactive input which you cannot provide.
+export COLOR_RED=$'\e[31m'
+export COLOR_YELLOW=$'\e[33m'
+export COLOR_RESET=$'\e[0m'
+
+if [ ! "$#" -eq 2 ]; then
+    echo "${COLOR_RED}You must provide exactly two argument!.${COLOR_RESET}"
+    exit 1
+fi
+
+# Param 1: Breeze Command
+# Param 2: Number of Retry(ies)
+COMMAND_TO_ATTEMPT="breeze $1"
+NUMBER_OF_ATTEMPT=$2
+NUMBER_OF_ATTEMPT="${NUMBER_OF_ATTEMPT:-1}"
+CURRENT_ATTEMPT=2
+ATTEMPT_SUCCESS=false
+
+breeze down
+set +e
+CURRENT_RUN_OUTPUT=$($COMMAND_TO_ATTEMPT)
+RESULT=$?
+echo "$CURRENT_RUN_OUTPUT"
+set -e
+
+while [ "${ATTEMPT_SUCCESS}" = false ] && [ "${CURRENT_ATTEMPT}" -le "${NUMBER_OF_ATTEMPT}" ]; do
```

Comment:
I think this script might be vastly simplified (many less environment variables and simpler steps by using for loop and passing all arguments of the script to the breeze command:

Something like that (with added diagnostics and printing of course) should ldo the job - might need some tweaking, as I am writing it from the top of my head :) :

```bash
for i in $(seq 1 "${NUM_ATTEMPTS}")
do
   if breeze "$@" then
      exit 0
   else
     RESULT=$?
   fi
done

# Error here
```

Reply:
Make sense! Let me apply the logic to the script. Thanks :)